app/views/courses/vacancies.html.erb

```diff
@@ -1,10 +1,61 @@
 <%= content_for :page_title, "Vacancies" %>
 
-<%= manage_ui_link_to "Back", "/organisation/#{params[:provider_code]}/courses/#{@course[:course_code]}", class: "govuk-back-link" %>
+<%= manage_ui_link_to "Back", "/organisation/#{params[:provider_code]}/courses/#{@course.course_code}", class: "govuk-back-link" %>
 
 <main role="main" class="govuk-main-wrapper" id="main-content">
-  <h1 class="govuk-heading-xl">
-    <span class="govuk-caption-xl"><%= @course[:name] %> (<%= @course[:course_code] %>)</span>
-    Edit vacancies
-  </h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_with url: provider_courses_path(@course.course_code) do |f| %>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset" aria-describedby="how-contacted-conditional-hint">
```

Comment:
Is there a reason we don't use the GovukElementsFormBuilder?

Reply:
It doesn't support govuk-frontend, only the legacy govuk-elements